src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -96,6 +97,22 @@ const VirtualAccOpening = (() => {
 
         if (Cookies.get('affiliate_tracking')) req.push({ request_field: 'affiliate_token', value: Cookies.getJSON('affiliate_tracking').t });
 
+        const mobile_devices = ['iPhone', 'iPad', 'Android'];
+        const date_first_contact = LocalStore.get('date_first_contact') != null ?
+            LocalStore.get('date_first_contact') :
+            toISOFormat(moment());
+        const signup_device = mobile_devices.some(item => item === navigator.platform) ? 'mobile' : 'desktop';
+
+        req.push({
+            request_field: 'signup_device',
+            value        : signup_device,
+        });
+
+        req.push({
+            request_field: 'date_first_contact',
+            value        : date_first_contact,
+        });
```

Comment:
why are we pushing these instead of directly including them in the object? there is no if wrapper etc